docs/src/main/tut/chisel3/modules.md

```diff
@@ -84,3 +84,68 @@ children modules, using the ```Module``` constructor function and
 the Scala ```new``` keyword to create a
 new object.  We then wire them up to one another and to the ports of
 the ```Mux4``` interface.
+
+### RawModule
+
+*Module* are used to connect components inside a design. But once we
+want to synthesize for the real world (FPGA, ASIC, ...) some extra
+features are required like io specific naming, reset polarity, clock
+assignement, ...
+
+For this, a specific module named *RawModule* should be used. With RawModule
+there is no implicit signals. All signals like clock and reset should be
+given in IO argument.
```

Comment:
I agree with Schuyler and would condense these two paragraphs into one. For example, we probably don't need to repeat something like `*Module* are used to connect components inside a design`
```
A `RawModule` is a module that allows you to define as much `IO` as needed with fully-custom naming (like `MultiIOModule`) but **does not provide an implicit clock and reset.** 
This can be useful when interfacing a Chisel module with a design that expects a specific naming convention for clock or reset.
```